imgui/src/context.rs

```diff
@@ -498,6 +508,7 @@ impl Context {
     /// # Panics
     ///
     /// Panics if the context uses a shared font atlas that is already borrowed
+    #[doc(alias = "NewFame")]
```

Comment:
Probably "Frame", but we're all searching for "Fame" I understand

Reply:
hehe